### DISCUSSION TITLE
Exploit causing excessive drive space usage.

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#941] [General]
# [ Subject of the issue ]
## Description
Describe your issue in as much detail as possible here. Excessive space needed for blockchain image due to exploit? What are the space requirements? 
![Screenshot_20221120-211616](https://user-images.githubusercontent.com/117119320/202926820-d14f18dc-bb54-47f1-819b-178c1c630421.png)

**Comment by laviniat1996** [comment]
Hi @Nexsuperne101 , may I ask from where did you get that quote, so there's a bit of context?
